wp-dependency-installer.php

```diff
@@ -622,6 +659,7 @@ public function hide_plugin_action_links( $plugin_file ) {
 			add_action(
 				'after_plugin_row_' . $plugin_file,
 				function ( $plugin_file ) {
+					print '<script>jQuery("tr[data-plugin=\'' . $plugin_file . '\'] .plugin-title > strong:first-child").append(\'<span class="wpdi-required"> *</span>\');</script>';
```

Comment:
Use jQuery to display an asterisk after required plugin name

Reply:
Why?

Reply:
Iit occurred to me because many forms use asterists to denote a mandatory field, it is not a too much intrusive modification but at the same time it helps to focus user attention on a non-standard behavior (and therefore to the `required plugin` label)